.travis.yml

```diff
@@ -1,26 +1,200 @@
+  - os: linux
+    python: 3.5
+    env: TOX_ENV=docs
+    before_install:
+      - sudo apt-get install python3.5
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 3.5
+    env: TOX_ENV=flake8
+    before_install:
+      - sudo apt-get install python3.5
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 2.7
+    env: TOX_ENV=py27
+    before_install:
+      - sudo apt-get install python2.7
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 3.4
+    env: TOX_ENV=py34
+    before_install:
+      - sudo apt-get install python3.4
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 3.5
+    env: TOX_ENV=py35
+    before_install:
+      - sudo apt-get install python3.5
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 3.6
+    env: TOX_ENV=py36
+    before_install:
+      - sudo add-apt-repository ppa:deadsnakes/ppa -y
+      - sudo apt-get update
+      - sudo apt-get install python3.6
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - sudo apt-get update -qq
+      - sudo locale-gen fr_FR.UTF-8 tr_TR.UTF-8
+
+  - os: linux
+    python: 3.7
+    env: TOX_ENV=py37
+    distro: xenial
+    before_install:
+      - sudo add-apt-repository ppa:deadsnakes/ppa -y
+      - sudo apt-get update
+      - sudo apt-get install python3.7
+      - python -m pip install virtualenv --user
+      - virtualenv venv
+      - source venv/bin/activate
+      - python -m pip install --upgrade pip wheel
+      - pip --version
+      - python -m pip install --upgrade pip wheel
+      - python -m pip install -r requirements/test.pip
+      - python -m pip install nose nose-cov coveralls pygments==2.1.3
```

Comment:
These should be `python3` and `pip3` I think. It's using 2.7 right now:
```
pip 18.1 from /home/travis/build/getpelican/pelican/venv/local/lib/python2.7/site-packages/pip (python 2.7)
```

Reply:
Now using the Python 3 version installed.

Turns out getting the Linux testing working again has been more involved that getting the Windows tests to work...